src/mesh/gmsh_reader.cpp

```diff
@@ -1663,16 +1685,37 @@ read_gmsh(std::string filename, int requested_grid_order)
 //        grid->output_results_vtk(9999);                    //Removed to save time (especially for large 3D grids) during flow solver test case. Grid will be outputted 'once' inside the gmsh_reader test file.
```

Comment:
remove or add a parameter for it

Reply:
Fixed this in the recent commit. Removed.